### DISCUSSION TITLE
千呼万唤始出来的K8s Sidecar

### DISCUSSION THREAD

**Discussion by giscus** [qingwave/qingwave.github.io#86] [Announcements]
# 千呼万唤始出来的K8s Sidecar

新版Sidecar使用与解析

https://qingwave.github.io/k8s-sidecar-released/

**Comment by yanhuan0802** [comment]
大佬，源码链接地址指向 https://github.com/kubernetes/kubernetes/pull/116 了，应该不太对

**Reply by qingwave** [reply]
多谢指正，正确的是https://github.com/kubernetes/kubernetes/pull/116429
